Exercise_1.go

```diff
@@ -0,0 +1,21 @@
+package main
+
+import "fmt"
+
+func UpdateArray(Array [5]int, index int, value int) {
```

Comment:
Please use idiomatic go - variables (and arguments) should always start with small letters. Also, you don't need long names in Go. In this case `a`, `i` and `v` would be absolutely enough:
```go
func Update(a [5]int, i int, v int) { a[i] = v }
```